net2/config.js

```diff
@@ -301,7 +349,11 @@ setInterval(() => {
   syncDynamicFeaturesConfigs()
 }, 60 * 1000) // every minute
 
+syncCloudFeaturesConfigs()
 
+setInterval(() => {
+  syncCloudFeaturesConfigs()
+}, 5 * 60 * 1000)
```

Comment:
no need to run this very frequently, it's okay to run only when firemain/fireapi starts. because if it changes very fast at runtime, it may cause internal data out of sync.